services/user.js

```diff
@@ -44,6 +44,31 @@ export default {
                         error: err
                     });
                 });
+        } else if (resource === 'user.socialsignin') {
+            rp.post({
+                uri: Microservices.user.uri + '/social/login',
+                body: JSON.stringify({
```

Comment:
I tried it but the service acted very strange. As the other calls to this service also use stringify I will not change it. Also the parameter "json: true" just do the stringify for you, thus no overhead.